configuration/security.md

```diff
@@ -92,6 +92,28 @@ alice ALL=(ALL) PASSWD: ALL
 
 Now save the file.
 
+
+## Disable or enable root login
+
+To disable root login, use:
+
+```bash
+sudo passwd -l root
+```
+
+To enable root login, use:
+
+```bash
+sudo passwd -u root
+```
+
+To enable root login again, use:
```

Comment:
I don't understand why there's "To enable root login" and "To enable root login again" ??

Reply:
By locking it, you're disabling it entirely. By enabling it again after unlocking it, you're setting the password.

Reply:
Sorry, I'm still confused - is there now a difference between "locking" & "disabling" and "enabling" & "unlocking" ?

I guess what I was asking originally (but perhaps not very clearly), was: should the "To enable root login" and "To enable root login again" sections be merged into one?

Reply:
> Sorry, I'm still confused - is there now a difference between "locking" & "disabling" and "enabling" & "unlocking" ?
> 
> I guess what I was asking originally (but perhaps not very clearly), was: should the "To enable root login" and "To enable root login again" sections be merged into one?

No, they do two different things. Once you lock and unlock the account, you've also gotta use passwd to reset the password.

Reply:
> Once you lock and unlock the account, you've also gotta use `$ sudo passwd root` to reset the password.

I think that's exactly what I was suggesting with:
> I guess what I was asking originally (but perhaps not very clearly), was: should the "To enable root login" and "To enable root login again" sections be merged into one?

:question: :confused:

Reply:
> Once you lock and unlock the account, you've also gotta use `$ sudo passwd root` to reset the password.

I think that's exactly what I was suggesting with:
> I guess what I was asking originally (but perhaps not very clearly), was: should the "To enable root login" and "To enable root login again" sections be merged into one?

:question: :confused: 

(your current instructions don't make it clear that after unlocking / enabling a previously-locked root account, you **also** need to change / reset the root password)

Reply:
Okay, I added clarification.